pkg/api/vlabs/types.go

```diff
@@ -519,6 +519,7 @@ type CustomCloudProfile struct {
 	AzureEnvironmentSpecConfig *AzureEnvironmentSpecConfig `json:"azureEnvironmentSpecConfig,omitempty"`
 	IdentitySystem             string                      `json:"identitySystem,omitempty"`
 	AuthenticationMethod       string                      `json:"authenticationMethod,omitempty"`
+	ConnectionMode             string                      `json:"connectionMode,omitempty"`
```

Comment:
My understanding is that field `ConnectionMode` groups together a bunch of base URLs.
If that's the case, the current field name does not reflect that fact immediately.

I may prefer slightly better something like `DependenciesSources` or `DependenciesLocation` with valid values being "default", "china", and "none".

Reply:
+1 

I didn't understand was `ConnectionMode` was at first when going over the code.